src/lib/desktop.ts

```diff
@@ -38,7 +38,7 @@ export async function renameProjectDirectory(
 
   // Make sure the new name does not exist.
   const newPath = window.electron.path.join(
-    projectPath.split('/').slice(0, -1).join('/'),
```

Comment:
jesus we need a lint for shit like this haha makes me nervous just looking at it